mopro-ffi/tests/bindings/test_mopro_keccak.swift

```diff
@@ -18,6 +18,31 @@ func bytesToBits(bytes: [UInt8]) -> [Int32] {
     return bits
 }
 
+// TODO: should handle 254-bit input
+func serializeOutputs(_ int32Array: [Int32]) -> [UInt8] {
```

Comment:
If this is a useful utility that is annoying to code in Swift, we could consider exposing it through FFI. What do you think? Maybe not in this PR but in general as part of "dealing with serialization stuff" https://github.com/oskarth/mopro/blob/main/mopro-core/src/middleware/circom/serialization.rs